ui-admin/src/forms/designer/questions/ChoicesList.tsx

```diff
@@ -0,0 +1,158 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronDown, faChevronUp, faPlus, faTimes } from '@fortawesome/free-solid-svg-icons'
+import React, { useId } from 'react'
+
+import { CheckboxQuestion, DropdownQuestion, RadiogroupQuestion } from '@juniper/ui-core'
+
+import { Button, IconButton } from 'components/forms/Button'
+import { TextInput } from 'components/forms/TextInput'
+
+type QuestionWithChoices = CheckboxQuestion | DropdownQuestion | RadiogroupQuestion
+
+type ChoicesListProps = {
+  question: QuestionWithChoices
+  readOnly: boolean
+  onChange: (newValue: QuestionWithChoices) => void
+}
+
+/** UI for editing the list of choices for a question. */
+export const ChoicesList = (props: ChoicesListProps) => {
+  const { question, readOnly, onChange } = props
+
+  const labelId = useId()
+
+  return (
+    <div className="mb-3">
+      <p className="mb-2" id={labelId}>Choices</p>
+      <ol aria-labelledby={labelId} className="list-group list-group-numbered mb-1">
```

Comment:
IMO the list numbers are distracting and might confuse users into thinking the choice numbers will appear in the participant UI.  I'd remove them

Reply:
I was thinking they might be useful for longer list of choices where otherwise it might require scrolling to get a sense of where you are in the list.

Reply:
Removed in https://github.com/broadinstitute/juniper/pull/444/commits/9592dad46617a41b586d13ffab58816d46636697.